config/ldap/init-ldap-acls.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -eu
+
+# load OpenLDAP environment and functions
+. /opt/bitnami/scripts/libopenldap.sh
+
+trap ldap_stop EXIT
+
+# start LDAP in background
+ldap_start_bg
+
+# wait until LDAP is started
+while ! ldapsearch -Y EXTERNAL -H ldapi:/// -b "cn=admin,dc=opencloud,dc=eu" >/dev/null 2>&1; do
+    echo "Waiting for LDAP to start..."
+    sleep 1
+done
+
+# apply acls
+echo -n "Applying acls... "
+ldapmodify -Y EXTERNAL -H ldapi:/// -f /opt/bitnami/openldap/etc/schema/50_acls.ldif
```

Comment:
I'd suggest using `slapmodify` here. That way you don't need to start the Server and could condense the whole script to:

```#!/usr/bin/env bash
set -eu

echo -n "Applying acls... "
slapmodify -F /opt/bitnami/openldap/etc/slapd.d -b cn=config -l /opt/bitnami/openldap/etc/schema/50_acls.ldif
if [ $? -eq 0 ]; then
    echo "done."
else
    echo "failed."
fi
```